library/src/main/java/com/google/android/exoplayer2/extractor/mkv/MatroskaExtractor.java

```diff
@@ -1403,6 +1405,7 @@ public void initializeOutput(ExtractorOutput output, int trackId) throws ParserE
           mimeType = MimeTypes.AUDIO_AAC;
           initializationData = Collections.singletonList(codecPrivate);
           break;
+        case CODEC_ID_MP2:
```

Comment:
I think it's fine/good to add MP2 to the list, but if you use `AUDIO_MPEG` then we'll end up with the mp3 decoder on regular Android devices being used to try and play mp2 streams, which will fail. I'm pretty sure you need to be using `AUDIO_MPEG_L2` instead for this. `FfmpegLibrary.getCodecName` appears to treat the two in the same way anyway, so changing it should "just work".

It's unclear to me whether `maxInputSize` should be set to anything or not. I'm not sure where `MP3_MAX_INPUT_SIZE` came from. @andrewlewis will know :).